src/OpenTracing/Carriers/TextMap.php

```diff
@@ -17,7 +17,11 @@ private function __construct(array $textMap)
         }
     }
 
-    public static function create(array $textMap = [])
+    /**
+     * @param array|string[] $textMap
+     * @return TextMap
+     */
+    public static function fromArray(array $textMap)
```

Comment:
@felixfbecker

Reply:
Probably not, we can add it later if people want it